libr/core/cconfig.c

```diff
@@ -3698,7 +3698,16 @@ R_API int r_core_config_init(RCore *core) {
 	SETPREF ("cfg.user", whoami, "set current username/pid");
 	free (whoami);
 	SETCB ("cfg.fortunes", "true", &cb_cfg_fortunes, "if enabled show tips at start");
-	SETCB ("cfg.fortunes.type", "tips,fun", &cb_cfg_fortunes_type, "type of fortunes to show (tips, fun)");
+	RList *fortune_types = r_core_fortune_types ();
+	RStrBuf *tmp_sb = r_strbuf_new ("");
+	RListIter *iter;
+	char *fortune_type;
+	r_list_foreach (fortune_types, iter, fortune_type) {
```

Comment:
use `char *fts = r_list_str_join(fortune_types, ",");` instead. it will avoid the heading comma and its just one line